src/PharoThings-Devices-HCSR04.package/PotHCSR04Device.class/README.md

```diff
@@ -0,0 +1,31 @@
+I'm a class to control the ultrasonic HCSR-04 sensors.
+You can use many ultrasonic sensors at the same time. Just change the PINs id. 
+
+To use:
+(inspector)
+ultrasonic := board installDevice: PotHCSR04Device new.
+
+(playground. change the board model  to your board)
+ultrasonic := (RpiBoard3B current) installDevice: PotHCSR04Device new.
```

Comment:
In playground it will be more complex:
```Smalltalk
board := RpiBoard3B current.
sensor := PotHCSR04Device triggerPin: (board pinWithId: 5) echoPin: (board pinWithId: 17).
board installDevice: sensor
```

Reply:
But we can simplify it a lot. For example:
```Smalltalk
RpiBoard3B current installDevice: (PotHCSR04Device triggerPin: 5 gpio echoPin: 17 gpio).
```
It will be equivalent of:
```Smalltalk
PotHCSR04Device triggerPin: (PotGPIOPin id: 5) echoPin: (PotGPIOPin id: 17).
```
There is already logic to sync device peripherals to preconfigured pins during installation process. So all "detached" pins in devices are replaced by ones from the board.

Reply:
This logic requires that device implements correctly #peripherals method.

Reply:
It was changed in the last PR.